Firebase/Messaging/FIRMessagingReceiver.m

```diff
@@ -19,7 +19,6 @@
 #import <UIKit/UIKit.h>
 
 #import <GoogleUtilities/GULAppEnvironmentUtil.h>
```

Comment:
Looks like we won't need this anymore either. We can't remove it from the podspec though because other classes use it.